rhel/Dockerfile

```diff
@@ -14,14 +14,13 @@ LABEL name="Kong" \
 
 COPY LICENSE /licenses/
 
-RUN yum install -y https://bintray.com/kong/kong-rpm/download_file?file_path=rhel/7/kong-$KONG_VERSION.rhel7.amd64.rpm && \
-    yum clean all && \
-    # OpenShift specific. OpenShift runs containers using an arbitrarily assigned user ID.
-    # This user doesn't have access to change file permissions during runtime, they have to be changed during image building.
-    # https://docs.okd.io/latest/creating_images/guidelines.html#use-uid
+RUN useradd kong && \
     mkdir -p "/usr/local/kong" && \
-    chgrp -R 0 "/usr/local/kong" && \
-    chmod -R g=u "/usr/local/kong"
+	chown -R kong:0 /usr/local/kong && \
+	chmod -R g=u /usr/local/kong && \
```

Comment:
The indentation is inconsistent here (tabs vs spaces). It's not going to break anything because it's part of a multi-line command, but it's going to look wacky if your editor is ever not configured so that 1 tab = 4 spaces (mine is 2 spaces).